### ISSUE TITLE
[typedoc-plugin-markdown@next] Bug: Links to `README.md` in header broken in member pages.

### ISSUE THREAD

**Issue by sharky98** [tgreyuk/typedoc-plugin-markdown#483]
Hello,

I have an issue where the links to the module `README.md` files are not relative from the member pages when building a doc for a monorepository (`"entryPointStrategy": "packages"`, based on [Gerrit0 example](https://github.com/Gerrit0/typedoc-packages-example)).

Version: `4.0.0-next.23(typedoc@0.25.1)`

I am using the default `outputFileStrategy` (`members`).

For example, taking the tree shown in the [option guide](https://github.com/tgreyuk/typedoc-plugin-markdown/blob/3a0a5bea81b14857d387fbfba599200212634eeb/packages/typedoc-plugin-markdown/docs/guides/options.md#--outputfilestrategy).

In the page `ClassA.md`, the header would uses `[README](README.md) ∙ [API](../modules.md)` for the links, correctly using the relative for the `modules.md` page, but not for the `README.md` page.

It is to note that the breadcrumbs do get relative links as expected, but they all point to the modules files, which already get its relative links correctly in the header.

`[API](../../../modules.md) > [package-1](../modules.md) > Content`

```
monorepo
├── README.md
├── modules.md
├── package-a
├── ├── README.md
├── ├── modules.md
│   ├── classes
│   │   ├── ClassA.md
│   │   ├── ClassB.md
├── ├── variables
│   │   ├── VariableA.md
│   │   ├── VariableB.md
├── package-b
├── ├── README.md
├── ├── modules.md
│   ├── classes
│   │   ├── ClassA.md
│   │   ├── ClassB.md
├── ├── variables
│   │   ├── VariableA.md
│   │   ├── VariableB.md
```

I was looking at the tests, and I'm not sure if I am reading this right, but it looks that this case is not covered. Seems that

- Generation of the `README.md` themselves are checked.

https://github.com/tgreyuk/typedoc-plugin-markdown/blob/3a0a5bea81b14857d387fbfba599200212634eeb/packages/typedoc-plugin-markdown/test/specs/__snapshots__/readmes.spec.ts.snap#L20-L48

- Existing links are checked. However, here I only see the `modules.md` links, not the readme (under both `package-1` and `package-2`, the URL is only for the `modules.md` and there is no `README.md` URLs.

https://github.com/tgreyuk/typedoc-plugin-markdown/blob/3a0a5bea81b14857d387fbfba599200212634eeb/packages/typedoc-plugin-markdown/test/specs/__snapshots__/navigation.spec.ts.snap#L3-L44

- Index page for a monorepo is checked.

https://github.com/tgreyuk/typedoc-plugin-markdown/blob/3a0a5bea81b14857d387fbfba599200212634eeb/packages/typedoc-plugin-markdown/test/specs/__snapshots__/groups.spec.ts.snap#L493-L509

- Urls are checked, and shows that `README.md` files are detected.

https://github.com/tgreyuk/typedoc-plugin-markdown/blob/3a0a5bea81b14857d387fbfba599200212634eeb/packages/typedoc-plugin-markdown/test/specs/__snapshots__/urls.spec.ts.snap#L113-L124

- When looking at the reflection tests, none seems to use the `packages` entry point strategy; but they do get relative URL to the `README.md` files.

**Comment by sharky98** [comment]
Could this be the culprit? 🧐

`function packageHeader()` not using `context.relativeURL()`

https://github.com/tgreyuk/typedoc-plugin-markdown/blob/3a0a5bea81b14857d387fbfba599200212634eeb/packages/typedoc-plugin-markdown/src/theme/resources/partials/header.ts#L86-L88

Compared to `function projectHeader()` using `context.relativeURL()`

https://github.com/tgreyuk/typedoc-plugin-markdown/blob/3a0a5bea81b14857d387fbfba599200212634eeb/packages/typedoc-plugin-markdown/src/theme/resources/partials/header.ts#L51-L53

**Comment by tgreyuk** [comment]
Thanks for feedback and the detailed description is very much appreciated. You were on the right lines about including the relativeUrl method, but the url needed to include the package directory.

Fix in typedoc-plugin-markdown@4.0.0-next.24 .
